CONTRIBUTING.md

```diff
@@ -1,57 +1,169 @@
-
-
 # Contributing to vcd-cli
 
-The *vcd-cli* project team welcomes contributions from the community. If you wish to contribute code and you have not
-signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
-questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
+Welcome! We gladly accept contributions from the community. If you wish
+to contribute code and you have not signed our contributor license
+agreement (CLA), our bot will update the issue when you open a pull
+request. For any questions about the CLA process, please refer to our
+[FAQ](https://cla.vmware.com/faq).
 
 ## Community
 
 https://vmwarecode.slack.com, #vcd channel
 
-## Contribution Flow
+## Logging Bugs
+
+Anyone can log a bug using the Github 'New Issue' button.  Please use
+a short title and give as much information as you can about what the
+problem is, relevant software versions, and how to reproduce it.  If you
+know the fix or a workaround include that too.
+
+## Code Contribution Flow
+
+We use Github pull requests to incorporate code changes from external
+contributors.  Typical contribution flow steps are:
+
+- Fork the vcd-cli repo into a new repo on Github
+- Clone the forked repo locally and set the original vcd-cli repo as the upstream repo
+- Make changes in a topic branch and commit them to your forked repo 
```

Comment:
I believe the flow is,
1. Work on feature branch,
2. Commit your work,
3. Fetch changes from upstream,
4. Resolve if necessary
5. Push commits to forked repo's feature branch
6. Submit a pull request to merge commits into vmware master.


Make changes in a topic branch and commit them _to your forked repo_ <-- this portion of the step shouldn't happen before merging upstream changes.